### ISSUE TITLE
Bug on Import (Android)

### ISSUE THREAD

**Issue by vitorpamplona** [tripleblindmarket/private-kit#114]
"Great idea. Unfortunately, the import from Google Location isn't working. I can get everything to download but then I get Error: -1"

This error was reported by Matthew Heflin on the PlayStore.

**Comment by manalive** [comment]
It's me, the Matthew Heflin. 

I'm on a Pixel 3XL running Android 9.

**Comment by conradm** [comment]
Same error here

Pixel 3xl
Android 10
Build: QQ2A.200305.002

![Screenshot_20200321-081146](https://user-images.githubusercontent.com/3872958/77221646-b9137200-6b4b-11ea-9fc6-744f95b19eec.png)

Creating a download works fine
Downloading a file works fine. 
After downloading a file I get the err - 1

**Comment by greggles** [comment]
I have this issue as well on a pixel3 google fi connected on wifi and Android 10. My location history goes back a few years and I selected the option to split by month and store the data in zip files in Google Drive.

I tried downloading the most recent month of data inside the iframe and I think that might have actually worked as I see a Download complete for a `.bin` file. However inside of PrivateKit i see the message in the screenshot from @conradm above: 

```
Error loading page 
Domain: undefined 
Error Code: -1
Description: net::ERR_FAILED
```

Looking through reviews in the google play store this is a pretty common error.
